docs/source/configuration/overview.mdx

```diff
@@ -203,4 +191,23 @@ headers:
       - insert: *insert_custom_header
 ```
 
-Here the content under `&insert_custom_header` would be reused under `*insert_custom_header`.
+Here, the `name` and `value` entries under `&insert_custom_header` are reused under `*insert_custom_header`.
+
+
+## Configuration awareness in your text editor
+
+The Apollo Router can generate a JSON schema for config validation in your text editor. This schema helps you format the YAML file correctly and also provides content assist.
+
+Generate the schema with the following command:
+
+```bash
+router --schema > configuration_schema.json
```

Comment:
Just for consistency with other examples of invoking the `router` (relativity, rather than global availability) 
```suggestion
./router --schema > configuration_schema.json
```